From 1f05fea77439f2ddd1db214a0119e624bf7146c9 Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Tue, 19 Jul 2016 05:03:52 +0000
Subject: [PATCH] HEAD-2308 download_metadata should retry fetch from SAPI on
 failure HEAD-2310 common RBAC setup should create "prof_attr.d" if missing
 HEAD-2311 sdc-scripts bootstrap shell library must tolerate "errexit", etc
 HEAD-2302 want support for rbac/pfexec access to zone metadata HEAD-2305
 adding files to prof_attr.d does not do anything TOOLS-1455
 sdc-scripts.git:sbin/postlogrotate.sh can attempt to cat log files over
 themselves HEAD-2307 HEAD-2302 needs to work on old PI (missing XML file)

---
 deps/sdc-scripts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index 9cd96a3..17fc5c8 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit 9cd96a35d84c223485e14f02e7a4ebca2088ca51
+Subproject commit 17fc5c8b83623078388b00260e5ad4be4f7348ac
-- 
2.21.0

